purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -33,6 +34,7 @@ import java.net.URL
 @Suppress("TooManyFunctions")
 class Purchases internal constructor(
     @get:JvmSynthetic internal val purchasesOrchestrator: PurchasesOrchestrator,
+    @get:JvmSynthetic internal val virtualCurrencyManager: VirtualCurrencyManager,
```

Comment:
I'm honestly not super excited about adding `virtualCurrencyManager` to the CEC Purchases object since virtual currencies aren't available in CEC mode, but given that the `virtualCurrencyManager` won't be used here and this is an internal constructor, I suppose it's okay?

I'm super open to suggestions on any ways to avoid adding this to the constructor here if you have them!

Reply:
Great idea!! I've implemented that here: b2448efd3833c8860ea65a86504cc4437bad497d